plugin/pkg/auth/authenticator/password/keystone/keystone.go

```diff
@@ -45,18 +45,18 @@ func (keystoneAuthenticator *KeystoneAuthenticator) AuthenticatePassword(usernam
 		glog.Info("Failed: Starting openstack authenticate client")
 		return nil, false, errors.New("Failed to authenticate")
 	}
-
+//FIXME user_id, not user_name...
 	return &user.DefaultInfo{Name: username}, true, nil
 }
 
 // NewKeystoneAuthenticator returns a password authenticator that validates credentials using openstack keystone
-func NewKeystoneAuthenticator(authURL string) (*KeystoneAuthenticator, error) {
-	if !strings.HasPrefix(authURL, "https") {
-		return nil, errors.New("Auth URL should be secure and start with https")
-	}
-	if authURL == "" {
-		return nil, errors.New("Auth URL is empty")
+func NewKeystoneAuthenticator(configPath string) (*KeystoneAuthenticator, error) {
+	if configPath != "" {
+		cfg, _, err := keystonetoken.LoadConfig(configPath)
```

Comment:
AuthenticatePassword need to be fixed to read and load the config file passed and override the username/password and authenticate.